modules/client/src/controllers/ConditionalTransferController.ts

```diff
@@ -1,7 +1,7 @@
-import EthCrypto from "eth-crypto";
 import { HashZero, Zero } from "ethers/constants";
 import { fromExtendedKey } from "ethers/utils/hdnode";
 
+import * as EthCrypto from "../eth-crypto";
```

Comment:
Which functions from the crypto module do we need? Specifying im/exports explicitly draws attention to the interface between this module and the rest of the codebase which encourages us to keep it as simple as possible which makes the code easier to maintain.